CanonicalLtd/ubuntu-image
# Flesh out basic project foundation

This branch add a bunch of basic project infrastructure including:

* Add a README
* Support coverage, qa, and testing via tox environments
* Add flake8 and nose2 extensions for qa and convenience
* Ensure minimum of Python 3.5
* Flake8 clean
* Reorganize test modules

Run the full test suite, including coverage and qa with `tox`.

For now, I've disabled `--fail-under=100` since we don't have 100% test coverage yet.  That seems premature during the exploratory phase, but eventually we need to enable that.
